rdr_service/cron_default.yaml

```diff
@@ -169,16 +159,6 @@ cron:
   timezone: America/New_York
   schedule: every saturday 22:00
   target: offline
-- description: Genomic Missing Files Resolve
-  url: /offline/GenomicMissingFilesResolve
-  timezone: America/New_York
-  schedule: every 6 hours
-  target: offline
-- description: Reconcile GC Data File to Table
-  url: /offline/ReconcileGCDataFileToTable
-  timezone: America/New_York
-  schedule: every tuesday 02:00
-  target: offline
```

Comment:
Same here with cleaning corresponding logic.

Reply:
Removed the code for these as well